src/Fable.Electron/Fable.Electron.fsproj

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Version>1.0.0</Version>
+    <Authors>Christer van der Meeren</Authors>
+    <Description>Fable bindings and helpers for Electron.</Description>
+    <PackageLicenseUrl>https://github.com/fable-compiler/fable-electron/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/fable-compiler/fable-electron</PackageProjectUrl>
+    <PackageTags>fable electron bindings</PackageTags>
+    <PackageReleaseNotes>Initial release</PackageReleaseNotes>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <WarningsAsErrors />
+  </PropertyGroup>
+  <PropertyGroup>
+    <NpmDependencies>
+      <NpmPackage Name="electron" Version=">= 6.0 lt 7" DevDependency="true" ResolutionStrategy="max" />
+    </NpmDependencies>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="paket.references" />
+    <Compile Include="Electron.fs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(Configuration)' == 'Release'">
+    <Content Include="*.fsproj; *.fs" PackagePath="fable\" />
```

Comment:
It's not necessary to add the `'$(Configuration)' == 'Release'` condition. It could break the library if you forget to add `-c Release` when packaging for publishing.

Reply:
BTW, if the library contains only bindings and not actual code you can omit this, which slightly improves the startup time when compiling. But it's probably a good idea to leave it to avoid problems in the future. Also I can see you one helper, `createAccelerator` (it may also be a good idea to put the helpers in a different file, this also helps the Fable because it can skip more code).

Reply:
The `Release` condition is there to avoid cluttering the file list in the Solution Explorer in VS (where it's mostly in Debug mode when developing). Release mode should always be used for releasing anyway. (I guess that for Fable, it makes no difference.)

Thanks for the helper tip, I'll move those to another file.

Reply:
Regarding the helpers in another file, could you briefly explain how this helps Fable?

Reply:
There are three steps in the compilation:

1. FSharp Compiler Services (FCS) takes F# code and transforms it to an AST
2. Fable takes the F# AST and transforms it to a Babe AST
3. Babel takes the AST and outputs JS code

Step 1 is done for all the F# files in the project (although I believe some operations are lazy), but step 2 and 3 are only done in demand. For consulting metadata (name and signature of the methods, attributes content...) step 2 is not needed. However, parsing is done at the file level. This means that if a user calls `createAccelerator`, Fable will have to go through the whole Fable.Electron.fs file even if only one function has actual code and the rest is metadata.

I don't have any benchmark, and maybe it doesn't matter too much (going through interface declarations should be fast). But given that Fable.Electron.fs allowing Fable to skip the "bindings" part may improve compilation time a bit, it may also help maintenance to separate the "semi-automatically" generated part, and the manually coded helpers.

Reply:
Great, thanks!

Reply:
Unfortunately I can't easily do that since I have the `Electron` module as a common top-level module for the bindings (directly in the `Electron` module) and the helpers (in a `Helpers` sub-module inside `Electron`).

I could have a top-level `Electron` namespace instead of module, but then I can't have the `main`/`renderer` entry points directly inside. I could perhaps place them in an auto-opened module inside the `Electron` namespace...

Do you have any tips on how I could organize this using modules/namespaces?

Reply:
Thanks! I've updated now. I think the library is more or less as it will be when released (only updating for any Electron API changes towards 6.0).